go.mod

```diff
@@ -65,13 +65,17 @@ require (
 	github.com/uber/jaeger-lib v2.4.1+incompatible // indirect
 	go.uber.org/atomic v1.11.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
-	golang.org/x/crypto v0.38.0 // indirect
+	golang.org/x/crypto v0.39.0 // indirect
 	golang.org/x/exp v0.0.0-20250305212735-054e65f0b394 // indirect
-	golang.org/x/net v0.40.0 // indirect
+	golang.org/x/net v0.41.0 // indirect
 	golang.org/x/oauth2 v0.30.0 // indirect
 	golang.org/x/sys v0.33.0 // indirect
-	golang.org/x/text v0.25.0 // indirect
+	golang.org/x/text v0.26.0 // indirect
 	google.golang.org/protobuf v1.36.5 // indirect
 	gopkg.in/go-jose/go-jose.v2 v2.6.3 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/cloudtrust/keycloak-client/v2 => ../keycloak-client
+
+replace github.com/cloudtrust/common-service/v2 => ../common-service
```

Comment:
For local testing. Will update the PR once keycloak-client and common-service are released.

Reply:
I understand it is very convenient for local testing. However it does not run in the CI. Do you know you can take module based on a branch ? this way will work in the CI (if the branch is pushed on github for sure)

for example : 
go get github.com/cloudtrust/common-service/v2@MyBranch

Reply:
Ah I didn't know, I fixed it.

Reply:
Better than a branch, you can use a commit (when pushing, you have a message with xxyyzzold...aabbccnew). Its easier with commit ids because if you push on a branch, go mod vendor does not see all changes